server/routes/gitlab/gitlab_routes.py

```
@@ -0,0 +1,287 @@
+            with conn.cursor() as cur:
+                cur.execute("SELECT org_id FROM users WHERE id = %s", (user_id,))
+                row = cur.fetchone()
+                org_id = row[0] if row else None
+    except Exception as e:
+        logger.warning(f"Could not fetch org_id for auto-connect: {e}")
+
+    count = 0
+    connected_names = []
+    try:
+        with db_pool.get_admin_connection() as conn:
+            with conn.cursor() as cur:
```

Ruff (0.15.12)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)